src/components/Table.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Table.css';
+
+const Row = ({ location }) => {
+  return (
+    <tr className="tableListing">
+      <td className="tableTitle">{location.title}</td>
+      <td className="tableDescription">{location.description}</td>
+      <td className="tableLink">
```

Comment:
Love the easy to follow classNames. 

Looks like the AC also requires the distance.